src/uu/cp/src/cp.rs

```diff
@@ -1454,22 +1454,36 @@ fn file_mode_for_interactive_overwrite(
 }
 
 impl OverwriteMode {
-    fn verify(&self, path: &Path, debug: bool) -> CopyResult<()> {
+    fn verify(&self, path: &Path) -> CopyResult<()> {
         match *self {
             Self::NoClobber => {
-                if debug {
-                    println!("skipped {}", path.quote());
-                }
+                // TODO
+                // Revert to 1_i32 when "mv/update" is fixed
```

Comment:
could you please add a link to the issue ?

Reply:
I need to make one. I remember that this is the part of the test causing problems, but I'll have to refresh my memory on this:

https://github.com/coreutils/coreutils/blob/8083944484f2cdf6c9b737642567bcdb54db784d/tests/mv/update.sh#L72

Reply:
please let me know when both comments are addressed :)